packages/web/src/public/locales/en/translation.json

```diff
@@ -1,4 +1,12 @@
 {
-  "languagesView": "this is where the languages view will go",
-  "translationView": "translation interface is work in progress"
+  "translationView": "translation interface is work in progress",
+  "language_one": "language",
+  "language_other": "languages",
```

Comment:
i18next is really well designed. You can define a translation key (ie: 'language'), with modifiers (ie '_one' or '_other' for plural modifiers). Then when you use the key you can pass in a count (id: `t('languages', { count: 10 })`), and it will pick the right modifier. Different languages can have different modifiers. So one language might define modifiers for 'zero', 'one', 'two', 'few' and 'many', while another might just have 'one', and 'many'.